src/main.rs

```diff
@@ -563,18 +572,33 @@ fn run_cli(cli: Cli) -> Result<()> {
                 }
             };
 
-            if edit {
+            if !no_alias {
+                match Alias::from_str(&id) {
+                    Ok(alias) => {
+                        info!("Creating alias '{alias}' for '{canonical}'");
+                        row.apply(state::add_alias(&alias))?;
+                    }
+                    Err(error::AliasConversionError::Empty) => {
+                        warn!("Alias not created for '{canonical}'.");
+                    }
```

Comment:
It is weird that this could happen. We should be blocking the creation of a `local:` record id if the chosen key is empty, I guess?

Reply:
I suppose this was actually a bug in the `RemoteId::local` function. Running `cargo run -- local ''` would have been successful before.

Fixed now, although I'm not completely happy with the `unreachable!` here. I wonder if there is a more satifying solution.

Reply:
Yeah fair enough, it looks good to me! I agree that the `unreachable` is a bit unsatisfying.

A potentially less arbitrary solution would be to require that `local:` keys always satisfy the alias rules.

Reply:
I.e. change the `local` initialization code to be infallible, but require passing an `Alias`.

Reply:
Right, this would make sense if we also insist that the `Alias` cannot exist already.